silkworm/core/protocol/validation.cpp

```diff
@@ -258,6 +254,17 @@ intx::uint256 compute_call_cost(const Transaction& txn, const intx::uint256& eff
     return required_funds;
 }
 
+intx::uint256 compute_call_required_funds(const Transaction& txn, const intx::uint256& effective_gas_price, const EVM& evm) {
```

Comment:
I think we at least need good documentation what this function is for and why it is different from `compute_call_cost()`.

Reply:
Function removed by @lupin012 in last refactoring